fga/model/model.fga

```diff
@@ -104,19 +104,25 @@ type control
     define can_view: [user] or can_view from parent
     define can_edit: [user] or can_edit from parent
     define can_delete: [user] or can_delete from parent
-    define parent: [program]
+    define parent: [user, service, program]
 type subcontrol
   relations
     define can_view: [user] or can_view from parent
     define can_edit: [user] or can_edit from parent
     define can_delete: [user] or can_delete from parent
-    define parent: [program]
+    define parent: [user, service, program]
 type control_objective
   relations
     define can_view: [user] or can_view from parent
     define can_edit: [user] or can_edit from parent
     define can_delete: [user] or can_delete from parent
-    define parent: [program]
+    define parent: [user, service, program]
+type risk
+  relations
+    define can_view: [user] or can_view from parent
+    define can_edit: [user] or can_edit from parent
+    define can_delete: [user] or can_delete from parent
+    define parent: [user, service, program]
```

Comment:
this should also include organization

Reply:
We are not directly inheriting the `organization` permissions (meaning having view access to the organization, does not automatically give you access to the risk). Instead I've added groups to allow for additional access and removed the requirement for a risk to have have to be associated to a program (and can be associated to 1:many programs instead of just 1). 

This means: 
- When a risk is created, if there are no programs or groups associated, only the user who created the risk can view/edit the risk
- If a risk is associated to program(s) permissions to the risk are inherited from the program, so if a user has edit access to the program, they will have edit access to the risk associated (same for view permissions)
- Risks can be associated with groups as editors, viewers, or blocked which allows for additional control over the access

Reply:
We are not directly inheriting the `organization` permissions (meaning having view access to the organization, does not automatically give you access to the risk). Adding the `organization` here as a `parent` would give this access. Instead I've added groups to allow for additional access and removed the requirement for a risk to have have to be associated to a program (and can be associated to 1:many programs instead of just 1). 

This means: 
- When a risk is created, if there are no programs or groups associated, only the user who created the risk can view/edit the risk
- If a risk is associated to program(s) permissions to the risk are inherited from the program, so if a user has edit access to the program, they will have edit access to the risk associated (same for view permissions)
- Risks can be associated with groups as editors, viewers, or blocked which allows for additional control over the access

However, I did add the org owned mixin, which requires an organization ID to be associated with the risk, and the user to have edit access to the organization to create the risk.